### ISSUE TITLE
Mod introduces Flint And Steel duplication.

### ISSUE THREAD

**Issue by Mirsario** [realguyman/totally_lit#37]
When Totally Lit `v0.13.7+mc1.20.1` is enabled, attempting to repair a Flint & Steel through combining two of them results in the components receiving 1 point of damage each, producing a new fully repaired Flint & Steel item. Uh oh!
Reproducible with nothing but dependencies.

**Expected behavior:**

https://github.com/user-attachments/assets/950a771c-4a79-41a0-b239-b74763ba598a

**Actual behavior:**

https://github.com/user-attachments/assets/b04b7480-c687-4d60-9a1a-d079823868a2

**Screenshots or videos:**

My entire server is now filled with flints & steels after my friend discovered this in secret.
![](https://github.com/user-attachments/assets/89d9a2e3-dd36-479f-9b8f-639311297d10)

**Comment by realguyman** [comment]
This feature has been removed with the latest release v0.15.1+1.21.1. Currently I no longer support Minecraft for any versions that aren't the latest, as I'm still in the development process for Totally Lit.

When I've finished the mod (1.0.0) I will begin backporting to older versions of Minecraft.

**Comment by Mirsario** [comment]
Thanks, I'll try to build a backport myself in the meanwhile.
On the topic of backports however, I recommend you to look into using a preprocessor like [Stonecutter](https://stonecutter.kikugie.dev) to target multiple versions from one branch and codebase. I recently switched my mod's [dev branch](https://github.com/Mirsario/Minecraft-CameraOverhaul/tree/dev) to it and it's pretty good. The syntax and manual state switching are less convenient than something like [Manifold-Preprocessor](https://github.com/manifold-systems/manifold), but the latter wasn't build for MC use-cases, causing me to run into some terrible Gradle plugin collision issues when I tried it first. Issues Stonecutter avoided entirely.
